1.5/Source/PawnStorages/PawnStorages/Farm/ITab_Slaughter.cs

```diff
@@ -75,12 +76,27 @@ public void DoMaxColumn(WidgetRow row, ref int val, ref string buffer, int curre
         }
     }
 
-    public void DrawLine(Rect rowRect, AutoSlaughterConfig config, int index)
+    public void DoAgeOrderColumn(WidgetRow row, ref bool ascending)
+    {
+        Texture2D tex = ascending ? PSContent.PS_ArrowUp : PSContent.PS_ArrowDown;
+        float width = (float) ((60.0 - 68.0) / 2.0);
```

Comment:
??? 60 - 68 / 2... you mean -4?
I'm assuming the intent here is to show the calculation because it's the label size of 60 and the 8 below etc but if that's true can we have them be named consts so it's clearer where this number comes from and so it auto updates if the 60 changes for whatever reason?

Reply:
Copy pasted from elsewhere. Must be a copy/paste of a copy/paste of some decompilation artifact.